Files/Filesystem/Search/FolderSearch.cs

```diff
@@ -31,26 +34,55 @@ internal class FolderSearch
 
         private uint UsedMaxItemCount => MaxItemCount > 0 ? MaxItemCount : uint.MaxValue;
 
-        public async Task<ObservableCollection<ListedItem>> SearchAsync()
+        public EventHandler SearchTick;
+
+        public string AQSQuery
         {
-            var results = new ObservableCollection<ListedItem>();
+            get
+            {
+                // if the query starts with a $, assume the query is in aqs format, otherwise assume the user is searching for the file name
+                if (Query is not null && Query.StartsWith("$"))
```

Comment:
I think this can be changed to `if (Query is not null && (Query.StartsWith("$") || Query.Contains(":")))`. Filenames don't contain ":" so if the user types ":" it's for sure AQS